lib/spack/spack/test/spec_syntax.py

```diff
@@ -286,6 +287,8 @@ def _check_hash_parse(self, spec):
             str(spec), spec.name + '@' + str(spec.version) +
             ' /' + spec.dag_hash()[:6])
 
+    @pytest.mark.skipif(sys.platform == 'win32',
+                        reason="Not supported on Windows (yet)")
```

Comment:
It looks like all tests in this module checking YAML and hash-based references fail. I think for spec parsing, we'll want to be careful to record what we do and don't support, since it is core to Spack's functionality.

Reply:
I'll add this onto the docket of test modules to re-enable. At the very least we'll have a better sense of it's failing and document that it remains unsupported on Windows.